src/types/tteokguk.ts

```diff
@@ -22,7 +22,9 @@ export interface UserTteokguk {
   completion: boolean;
   access: boolean;
   tteokgukIngredients: IngredientKey[];
-  backgroundColor: "BLUE" | "GREEN" | "PINK" | "YELLOW";
+  backgroundColor: BackgroudColor;
   frontGarnish: IngredientKey;
   backGarnish: IngredientKey;
 }
+
+export type BackgroudColor = "BLUE" | "GREEN" | "PINK" | "YELLOW";
```

Comment:
```suggestion
export type TteokgukBackgroudColor = "BLUE" | "GREEN" | "PINK" | "YELLOW";
```

TteokgukBackgroudColor가 네이밍이 더 적절해 보입니다~